src/js/tabs/osd.js

```diff
@@ -2985,6 +2983,14 @@ osd.initialize = function(callback) {
                     // Hide custom if not used
                     $('.osdfont-selector option[value=-1]').toggle(osdFontSelectorElement.val() === -1);
 
+                    // Zoom option for the preview only for mobile devices
+                    if (GUI.isCordova()) {
+                        $('.osd-preview-zoom-group').css({display: 'inherit'});
```

Comment:
Why not just using css media query for that classname instead of js?

Reply:
The media query was my first attempt, using resolution, but finally I wanted to restrict to Android only. Is a way in the media query to limit to Android specifically?